entrypoints/popup/router.tsx

```diff
@@ -224,6 +227,14 @@ export const router = createHashRouter([
                 path: "connect-ledger",
                 element: <LedgerConnect />,
               },
+              {
+                path: "deploy-token",
+                element: <DeployToken />,
+              },
+              {
+                path: "mint-token",
+                element: <DeployToken />,
```

Comment:
`mint-token` should be another page from my side.

Reply:
b85867cd9b85406f6f35e3e8209ab021076bea4f